peps/pep-0772.rst

```diff
@@ -0,0 +1,1612 @@
+to install a Python runtime. Many, likely most, are interested in running
+(perhaps writing) short scripts, such as those that perform a simple task,
+or help teach someone a concept. Some users are looking for a specific version
+to integrate with existing code or another application. Some are after a full
+set of different interpreter versions to perform testing.
+
+In this section, we discuss the expectations that users have of "installing
+Python", provide an overview of the existing installers for Windows, and
+identify some of the gaps and challenges inherent in these offerings.
+
+Expectations
+------------
+
+Based on significant anecdotal experience and analysis of quantitative data
+available (though not necessarily public), we make the following assertions
+about the majority of Python users on Windows:
+
+* most users just want the latest stable version
+* most users want a "one-click" (or fewer) install
+* most users do not want to use administrator privileges
+* most users will benefit from installing maintenance updates
+* most users expect the ``python`` command to work after installation
+
+The primary support for these assertions is that the most popular installers
+actively chosen by users are the latest stable release on python.org, and
+the latest stable release on the Windows Store, both of which meet these
+requirements.
+
+We make the following assumptions about other significant sets of users.
+These may have some overlap between groups, and at least some users expect
+all of them.
+
+* some users want to install Python programmatically
+* some users want to install a particular version
+* some users want to install many versions
+* some users want to install for all users of their machine
+* some users do not want Start Menu shortcuts
+* some users want to install as part of their project's build process
+* some users want to install as part of their project's install process
+* some users intend to never update their install
+* some users expect the ``py`` command to work after installation
+
+These assumptions have all been demonstrated over time to exist, though the
+relative importance has not been quantified. The Nuget packages and the
+embeddable distro can meet most of these needs.
+
+Traditional Installer
+---------------------
+
+The traditional installer is an executable downloadable directly from
+python.org that installs the entire development kit for Python. This includes
+the CPython interpreter, the standard library, Python headers and import
+libraries, builds of Tcl and Tk, the documentation as HTML files, the runtime
+and standard library test suite, Start Menu shortcuts for Python and IDLE,
+debugging symbols and debug builds of the binaries, the ``py.exe`` launcher
+and its file associations, and functionality to modify the user's ``PATH``
+environment variable, enable long-path support on their system, pre-generate
+``.pyc`` files for the standard library, and install pip.
```

Comment:
```suggestion
``.pyc`` files for the standard library, and a ``pip`` installation.
```

Reply:
pip doesn't quote their own name, so I chose not to.